console-subscriber/src/builder.rs

```diff
@@ -176,6 +180,15 @@ impl Builder {
         }
     }
 
+    /// Sets whether tasks, resources, and async ops from the console 
+    /// subscriber thread are recorded. 
```

Comment:
gah, rustfmt is mad that i accidentally added trailing whitespace here
```suggestion
    /// subscriber thread are recorded.
```